configs/tasks/rearrangepick_replica_cad.yaml

```diff
@@ -3,7 +3,7 @@ ENVIRONMENT:
 DATASET:
     TYPE: RearrangeDataset-v0
     SPLIT: train
-    DATA_PATH: data/datasets/rearrange_pick/replica_cad/v0/{split}/{split}_counter_L_analysis_5000_500.json.gz
+    DATA_PATH: data/datasets/rearrange_pick/replica_cad/v0/rearrange_pick_replica_cad_v0/{split}/{split}_counter_L_analysis_5000_500.json.gz
```

Comment:
We should use the `tidy_house_10k_1k` dataset. That is a very old dataset and does not support proper goal placements.

Reply:
This file is not used by Galakhtic. Still, we should probably rebase Gala's branch of hab-lab to something near latest (we'll do this in a separate PR).